src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.ilproj

```diff
@@ -4,6 +4,7 @@
     <ProjectGuid>{04BA3E3C-6979-4792-B19E-C797AD607F42}</ProjectGuid>
     <IncludePath>include\$(TargetGroup)</IncludePath>
     <IncludePath Condition="'$(TargetGroup)' == 'netcoreapp2.0'">include\netcoreapp</IncludePath>
+    <IncludePath Condition="'$(TargetGroup)' == 'netfx'">include\net461</IncludePath>
```

Comment:
shouldnt this condition be changed to ```TargetsNetFx```` as we will want to include this file for net461 and net472. This will only include the file for net472.

Reply:
Not really, if you see line 5, we always automatically add "include\$(TargetGroup)" which for net461 build will include the file. We didn't want to duplicate the file, which is why we add this extra condition for the netfx case, so that it includes the same file as net461 build.